android/src/main/java/com/swmansion/gesturehandler/RNGestureHandlerPackage.kt

```diff
@@ -71,7 +71,7 @@ class RNGestureHandlerPackage : TurboReactPackage(), ViewManagerOnDemandReactPac
             RNGestureHandlerModule::class.java.name,
             reactModule.canOverrideExistingModule,
             reactModule.needsEagerInit,
-            reactModule.hasConstants,
+            true, // Has constants is hardcoded to return true, so replacing it with `true` changes nothing.
```

Comment:
While leaving this line still leads to deprecation warning during build phase, we decided to leave that as it is, because removing this parameter leads to compilation errors on lower RN versions.